File: PYTHON/Kode/Clothing.py
class Clothing:
    def __init__(self, brand, size, price):
        self.brand = brand
        self.size = size
        self.price = price
    
    def get_brand(self): 
        return self.brand
    
    def get_size(self): 
        return self.size
    
    def get_price(self): 
        return self.price
    
    def set_brand(self, brand): 
        self.brand = brand

    def set_size(self, size): 
        self.size = size
        
    def set_price(self, price): 
        self.price = price
    
    def display(self):
        return f"| {self.brand:<15} | {self.size:<10} | Rp {self.price:<10,.2f} |"

File: C++/Kode/Clothing.cpp
#ifndef CLOTHING_H //agar tidak terjadi tabrakan atau bahkan error, soalnya dari tadi error terus
#define CLOTHING_H

#include <iostream>
#include <iomanip>
#include <string>

using namespace std;

class Clothing {
private:
    string brand;
    string size;
    double price;

public:
    Clothing(string brand, string size, double price) {
        this->brand = brand;
        this->size = size;
        this->price = price;
    }
    
    void setBrand(string brand) { 
        this->brand = brand; 
    }

    void setSize(string size) { 
        this->size = size; 
    }

    void setPrice(double price) { 
        this->price = price; 
    }
    
    string getBrand() { 
        return brand; 
    }

    string getSize() { 
        return size; 
    }

    double getPrice() { 
        return price; 
    }
    
    virtual void display() {
        cout << "| " << left << setw(15) << brand << " | " << setw(10) << size << " | Rp " << setw(10) << fixed << setprecision(2) << price;
    }
};

#endif